webapp/pages/index.tsx

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2022 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+const Index = () => {
+  // TODO(ikhoon):
+  //  - Move pages folder under 'src/' directory. https://nextjs.org/docs/advanced-features/src-directory
```

Comment:
For better folder organization, it would be worth moving `pages/` directory into `src/pages/`